### ISSUE TITLE
Report user position to server

### ISSUE THREAD

**Issue by niklasmohrin** [hpi-swt2/compass-portal-blue#118]
In order to share the position with others, we first need to tell the server about it.

Requirements:
- [ ] the server has some internal state that holds the most recent position pings from clients that opted in to send their location

**Comment by JulianEgbert** [comment]
Maybe this should be saved in the user (person?) database as well instead of a new database. The accessibility of this value however should then be restricted.

**Comment by niklasmohrin** [comment]
> Maybe this should be saved in the user (person?) database as well instead of a new database. The accessibility of this value however should then be restricted.

If we would have to that info over restarts, I would agree, but:

If the location pings come in regularly, it should suffice to keep them in a Ruby Singleton somewhere. Its probably even better to not persist location data in the DB for privacy concerns - we should still clean that object regularly
